cfme/generic_objects/instance/__init__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+import attr
+
+import sentaku
+
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.update import Updateable
+
+
+@attr.s
+class GenericObjectInstance(BaseEntity, Updateable, sentaku.modeling.ElementMixin):
+    """Generic Objects class to context switch between REST and Automate.
+
+    Read/Update/Delete functionality.
+    """
+    update = sentaku.ContextualMethod()
+    delete = sentaku.ContextualMethod()
+    exists = sentaku.ContextualProperty()
+
+    name = attr.ib(default=None)
+    definition = attr.ib(default=None)  # generic object definition
+    attributes = attr.ib(default=None)  # e.g. {'address': 'Test Address'}
+    associations = attr.ib(default=None)  # e.g. {'services': [myservice1, myservice2]}
+
+    def __attrs_post_init__(self):
+        self.rest_response = None
```

Comment:
**REQUIRED:** Why do we need to use ```__attrs_post_init__``` here? If you don't want it to be in the constructor, attrs has an option for that